app/views/partials/transferAgreementFormGroup.scala.html

```diff
@@ -6,27 +6,35 @@
 
 @(transferAgreementFormData: Form[_],
   taFormNamesAndLabels: Seq[(String, String)],
-  formAlreadySubmitted: Boolean
+  formAlreadySubmitted: Boolean,
+  formHeading: String
```

Comment:
Maybe change this param name to match what it is, eg: `fieldSetLegend: String`

Reply:
Changed `formHeading` to `fieldSetLegend` everywhere, thanks.